dockerfiles/deb.dockerfile

```diff
@@ -125,11 +125,11 @@ RUN ctr --version
 RUN runc --version
 
 FROM scratch AS packages
-COPY --from=build-packages  /archive /archive
-COPY --from=verify-packages /build   /build
+COPY --link --from=build-packages  /archive /archive
+COPY --link --from=verify-packages /build   /build
```

Comment:
The first `--link` probably doesn't do anything as we _start_ from `scratch` already, but I added it to keep nice formatting 😅